### PR TITLE
Create gk-web.md

### PR DESCRIPTION
**Pull request by ghost**
https://raw.githack.com/gk-web/gk-web.github.io/git-newcode/#!pages/vi/profiles/gk-web.md

### CODE REVIEW

File: pages/vi/profiles/gk-web.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+* I am Gautam Khanna situated in Waterloo, ON, Canada {⏰: Eastern Daylight Time(EDT)}
+* I am currently a graduate student of *ECE* in the **University of Waterloo**
+* The OS I am working for is macOS Catalina (Version 10.15)
+* My hobbies are:
+    1. Programming
+       1a. C/C++
```
Comment (CalebProvost) [inline pages/vi/profiles/gk-web.md:9]: 4th markdown style:
Numeric points

You need just one more markdown style added to your profile. Maybe a header? Image? Link? Its up to you.

Good work btw. Almost done with step. Keep up the good work! :smile:

File: pages/vi/profiles/gk-web.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+* I am Gautam Khanna situated in Waterloo, ON, Canada {⏰: Eastern Daylight Time(EDT)}
+* I am currently a graduate student of *ECE* in the **University of Waterloo**
```
Comment (CalebProvost) [inline pages/vi/profiles/gk-web.md:4]: 3 markdown styles found so far:
bullet point, italics, bold

**Review by CalebProvost — CHANGES_REQUESTED** [review]

File: pages/vi/profiles/gk-web.md (hunk 1 above)
Comment (ghost) [reply pages/vi/profiles/gk-web.md:9]: Hi, I have added my URL. Please have a look. Thanks!

Comment (CalebProvost) [reply pages/vi/profiles/gk-web.md:9]: Almost. You can see how to make links [here](https://guides.github.com/features/mastering-markdown/#example-text).
Just needs four more characters.

Comment (ghost) [reply pages/vi/profiles/gk-web.md:9]: Is there no link on the profile?

Comment (CalebProvost) [reply pages/vi/profiles/gk-web.md:9]: Only as text, not as markdown clickable link.

Comment (ghost) [reply pages/vi/profiles/gk-web.md:9]: Oh Sorry! My bad. Silly mistake!

Comment (ghost) [reply pages/vi/profiles/gk-web.md:9]: Done

Comment (gk-web) [reply pages/vi/profiles/gk-web.md]: Does it take time to see the changes I’ve made?

Comment (ghost) [reply pages/vi/profiles/gk-web.md:9]: Like the edits I make, do they reflect on the profile after some time?

Comment (CalebProvost) [reply pages/vi/profiles/gk-web.md:9]: Almost. Taken from the link I sent you above:

Learn more | [here](https://guides.github.com/features/mastering-markdown/)
------------ | -------------
It's very easy to make some words \*\*bold\*\* and other words \*italic\* with Markdown. You can even \[link to Google!\]\(http://google.com\) | It's very easy to make some words **bold** and other words *italic* with Markdown. You can even [link to Google!](http://google.com)

Comment (CalebProvost) [reply pages/vi/profiles/gk-web.md:9]: Your link is being rendered through html so far.
You'll need to adjust it to be a [markdown link](https://github.com/treehouses/treehouses.github.io/pull/224#discussion_r335629676) in order for it to count.

Comment (ghost) [reply pages/vi/profiles/gk-web.md:9]: Changed. please have a look.

**Review by CalebProvost — APPROVED** [review]

**Comment by CalebProvost** [comment]
Don't forget to request reviews from the [gitter chat](https://gitter.im/treehouses/Lobby)

**Review by XavierElon — COMMENTED** [review]
You need to use more markdown elements before we can approve this

**Comment by CalebProvost** [comment]
> You need to use more markdown elements before we can approve this

He meets the requirements
Requirements:
> Please use at least 5 different types of Markdown element

His page:

* 1st: `* I am Gautam Khanna situated in Waterloo, ON, Canada {⏰: Eastern Daylight Time(EDT)}`
  * it has bullet point
* 2nd and 3rd: `* I am currently a graduate student of *ECE* in the **University of Waterloo**`
  * it has bold and italics
* 4th: `* [LINK to LinkedIn Profile](http://www.linkedin.com/in/gautam-khanna-811942141/)`
  * it has markdown linking
* 5th and 6th has numeric listing and sub-listing:
```
* My hobbies are:
    1. Programming
       1a. C/C++
```

He meets the minimum requirement of 5 markdown styles.

**Comment by XavierElon** [comment]
> > You need to use more markdown elements before we can approve this
> 
> He meets the requirements
> Requirements:
> 
> > Please use at least 5 different types of Markdown element
> 
> His page:
> 
>     * 1st: `* I am Gautam Khanna situated in Waterloo, ON, Canada {⏰: Eastern Daylight Time(EDT)}`
>       
>       * it has bullet point
> 
>     * 2nd and 3rd: `* I am currently a graduate student of *ECE* in the **University of Waterloo**`
>       
>       * it has bold and italics
> 
>     * 4th: `* [LINK to LinkedIn Profile](http://www.linkedin.com/in/gautam-khanna-811942141/)`
>       
>       * it has markdown linking
> 
>     * 5th and 6th has numeric listing and sub-listing:
> 
> 
> ```
> * My hobbies are:
>     1. Programming
>        1a. C/C++
> ```
> 
> He meets the minimum requirement of 5 markdown styles.

Okay, didn't look like it at first glance

**Review by XavierElon — APPROVED** [review]

**Comment by ghost** [comment]
I will put some more elements to make my profile look good.

**Review by dogi — APPROVED** [review]

**Pull request merged**